cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+# 
+# Test locally:
+#
+#    gcloud builds submit --verbosity info --substitutions _STAGING_PROJECT=<your-project>,_PULL_BASE_REF=master [SOURCE]
+#
+timeout: 1800s
+options:
+  substitution_option: ALLOW_LOOSE
+steps:
+- name: gcr.io/k8s-testimages/gcb-docker-gcloud:v20200421-a2bf5f8
+  entrypoint: hack/release.sh
+  env:
+  - REGISTRY=gcr.io/${_STAGING_PROJECT}
+  - ALL_ARCH=${_ALL_ARCH}
+  - ALLOW_DIRTY=yes
+  - ALLOW_UNSTABLE=yes
+  - CONFIRM=yes
+  - SKIP_PUSH_LATEST=yes
```

Comment:
How will we support the canary job? https://k8s-testgrid.appspot.com/sig-storage-local-static-provisioner#master-gce-lastest

Reply:
Oh I see canary is different from latest